docs/source/customizations/native.mdx

```diff
@@ -1,33 +1,33 @@
 ---
-title: Writing Native Rust Plugins for the Apollo Router
-subtitle: Extend the Apollo Router with custom Rust code
-description: Extend the Apollo Router with custom Rust code. Plan, build, and register plugins. Define plugin configuration and implement lifecycle hooks.
+title: Writing Native Rust Plugins for the GraphOS Router
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)